mate_bot/commands/help.py

```diff
@@ -16,12 +16,9 @@ class HelpCommand(BaseCommand):
     """
 
     def __init__(self):
-        super().__init__(
-            "help",
-            "The `/help` command prints the help page for any "
-            "command. If no argument is passed, it will print its "
-            "usage and a list of all available commands."
-        )
+        super().__init__("help", "The `/help` command prints the help page for any "
+                                 "command. If no argument is passed, it will print its "
+                                 "usage and a list of all available commands.")
```

Comment:
This re-formatting seems to be useless. Most other files still use the previous way of indentation (e.g. `communism.py` and `vouch.py`). I prefer to not change it here.

Reply:
i checked only one command file and it was formatted like this, so i used this formatting. But i like it the otherway too.